src/generator_rom.rs

```diff
@@ -50,6 +50,28 @@ pub const GENERATOR_ROM: [u8; 737] = [
     0x80,
 ];
 
+// https://github.com/Chia-Network/chia-blockchain/blob/main/chia/wallet/puzzles/rom_bootstrap_generator2.clsp.hex
+
+pub const GENERATOR2_ROM: [u8; 265] = [
+    0xff, 0x02, 0xff, 0xff, 0x01, 0xff, 0x02, 0xff, 0x08, 0xff, 0xff, 0x04, 0xff, 0x02, 0xff, 0xff,
```

Comment:
This construct has always bothered me and I just ran across this crate which we could use at compile-time only to tidy it up a bit:

https://docs.rs/hex-literal/latest/hex_literal/

Maybe even just import the `.hex` file and use it with `include_str!` or `include_bytes!`.